site/de/index.html

```diff
@@ -0,0 +1,664 @@
+                            <pre  class="prettyprint lang-java">
+@Test
+public void getFirstNamesOfAllPeople()
+{
+  MutableList&lt;Person&gt; people = this.people;
+  MutableList&lt;String&gt; firstNames = people.collect(Person::getFirstName);
+  MutableList&lt;String&gt; expectedFirstNames =
+    Lists.mutable.with("Mary", "Bob", "Ted", "Jake", "Barry", "Terry", "Harry", "John");
+  Assert.assertEquals(expectedFirstNames, firstNames);
+}</pre>
+                        </div>
+                    </div>
+                    <!-- /.tab-content -->
+                </div>
+                <!-- /.col -->
+            </div>
+            <!-- /.row -->
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- Kata -->
+
+    <a class="anchor" name="history"></a>
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <h2 class="section-heading">Geschichte der Eclipse Collections</h2>
+            <p class="lead">
+                Die Basis der Eclipse Collections war ein Framework namens Caramel, welches 2004 bei Goldman Sachs entstand.
+
+                Das Framework entwickelte sich durch aktive Nutzung weiter und wurde 2012 als Open Source Projekt als GitHub
+                Projekt unter dem Namen <a href="https://github.com/goldmansachs/gs-collections">GS Collections</a> veröffentlicht.
+                <br><br>
+                GS Collections wurde danach bei einer Reihe von
+                <a href="https://github.com/eclipse/eclipse-collections/wiki/Conference-talks-and-meetups">Konferenzen</a>,
+                inklusive dem JVM Summit 2012 und der JavaOne in 2014, präsentiert.
+                Ein Geschwindigkeitsvergleich zwischen den parallelen Collections von Java 8, Scala und GS Collections wurde auf der
+                <a href="http://www.infoq.com/presentations/java-streams-scala-parallel-collections">QCon New York</a> in 2014 präsentiert.
+                Auch <a href="https://github.com/eclipse/eclipse-collections/wiki/Articles">Artikel</a> über die GS Collections
+                (<a href="http://www.infoq.com/articles/GS-Collections-by-Example-1">Teil 1</a> /
+                <a href="http://www.infoq.com/articles/GS-Collections-by-Example-2">Teil 2</a>)
+                wurden auf InfoQ.com veröffentlicht, welche die Fähigkeiten des Collections Framework durch Beispiele aufzeigen.
+                Weiterhin sind Interviews mit den Entwicklern der GS Collections vorhanden.
+                <br><br>
+
+                Über die Jahre haben sich rund 40 Entwickler an der Entwicklung der Eclipse Collections beteiligt.
+                <br><br>
+
+                Zur Vervollständigung des Open Source Gedankens wurden die GS Collections in 2015 der Eclipse Foundation
+                übertragen und in Eclipse Collections umbenannt.
+                Damit ist das Collections Framework heute komplett Open Source und erwartet deine Mitarbeit.
```

Comment:
und erwartet deine Mitarbeit.
=>
und freut sich auf deine Mithilfe.
(in the sense of "... is happily looking forward for your help")